[rospy.client][INFO] 2016-06-04 16:24:30,611: init_node, name[/script_server], pid[18600]
[xmlrpc][INFO] 2016-06-04 16:24:30,611: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2016-06-04 16:24:30,612: Started XML-RPC server [http://cmeintel-desktop:42352/]
[rospy.init][INFO] 2016-06-04 16:24:30,612: ROS Slave URI: [http://cmeintel-desktop:42352/]
[rospy.impl.masterslave][INFO] 2016-06-04 16:24:30,612: _ready: http://cmeintel-desktop:42352/
[rospy.registration][INFO] 2016-06-04 16:24:30,613: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2016-06-04 16:24:30,613: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2016-06-04 16:24:30,712: registered with master
[rospy.rosout][INFO] 2016-06-04 16:24:30,712: initializing /rosout core topic
[rospy.rosout][INFO] 2016-06-04 16:24:30,715: connected to core topic /rosout
[rospy.simtime][INFO] 2016-06-04 16:24:30,716: initializing /clock core topic
[rospy.simtime][INFO] 2016-06-04 16:24:30,719: connected to core topic /clock
[rosout][INFO] 2016-06-04 16:24:30,735: script_server is running
[rospy.internal][INFO] 2016-06-04 16:24:31,002: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2016-06-04 16:24:32,172: topic[/clock] adding connection to [http://cmeintel-desktop:35019/], count 0
[rospy.core][INFO] 2016-06-04 16:28:00,740: signal_shutdown [signal-2]
[rospy.internal][INFO] 2016-06-04 16:28:00,744: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2016-06-04 16:28:00,744: topic[/clock] removing connection to http://cmeintel-desktop:35019/
[rospy.topics][ERROR] 2016-06-04 16:28:00,745: Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 307, in close
    c.close()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 807, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2016-06-04 16:28:00,745: signal-2
[rospy.internal][WARNING] 2016-06-04 16:28:00,745: Unknown error initiating TCP/IP socket to cmeintel-desktop:37193 (http://cmeintel-desktop:35019/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2016-06-04 16:28:01,241: signal_shutdown [atexit]
